src/Stratis.Bitcoin.Features.Wallet/Events/TransactionFound.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Stratis.Bitcoin.EventBus;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.Wallet.Events
+{
+    /// <summary>
+    /// Event that is executed when a transaction is found in the wallet.
+    /// </summary>
+    /// <seealso cref="Stratis.Bitcoin.EventBus.EventBase" />
+    public class TransactionFound : EventBase
```

Comment:
@zeptin I can't seem to see who or what listens for this event? I can see it being published but no listener?

Reply:
Yeah true, I've removed it